rundeckapp/src/main/groovy/org/rundeck/app/data/providers/GormWebhookDataProvider.groovy

```diff
@@ -46,65 +46,71 @@ class GormWebhookDataProvider implements WebhookDataProvider {
 
     @Override
     Webhook findByUuidAndProject(String uuid, String project) {
-        return Webhook.findByUuidAndProject(uuid, project);
+        return Webhook.findByUuidAndProject(uuid, project)
     }
 
     @Override
     Webhook findByName(String name) {
-        return Webhook.findByName(name);
+        return Webhook.findByName(name)
     }
 
     @Override
     List<Webhook> findAllByProject(String project) {
-        return Webhook.findAllByProject(project);
+        return Webhook.findAllByProject(project)
     }
 
     @Override
     List<Webhook> findAllByNameAndProjectAndUuidNotEqual(String name, String project, String uuid) {
-        return Webhook.findAllByNameAndProjectAndUuidNotEqual(name, project, uuid);
+        return Webhook.findAllByNameAndProjectAndUuidNotEqual(name, project, uuid)
     }
 
     @Override
     Integer countByAuthToken(String authToken) {
-        return Webhook.countByAuthToken(authToken);
+        return Webhook.countByAuthToken(authToken)
     }
 
     @Override
     Integer countByNameAndProject(String name, String project) {
-        return Webhook.countByNameAndProject(name, project);
+        return Webhook.countByNameAndProject(name, project)
     }
 
     @Override
     void delete(Long id){
-        webhookDataService.delete(id);
+        Webhook.withTransaction {
```

Comment:
Shouldn't these be wrapped in a try/catch that throws DataAccessException like AuthToken above?

Reply:
I'll add those cases